app/cdap/components/WrangleHome/Components/OngoingDataExplorations/utils.ts

```diff
@@ -39,9 +51,9 @@ export const generateDataForExplorationCard = (oldData) => {
           obj.label = `${eachItem[keys]} Recipe steps`;
           obj.type = 'text';
         } else if (keys === 'dataQuality') {
-          obj.label = parseInt(eachItem[keys], 2);
+          obj.label = Number(eachItem[keys]);
           obj.percentageSymbol = '%';
-          obj.subText = 'Data Quality';
+          obj.subText = 'Null values';
```

Comment:
put the strings in constant file and import for checking in conditions

Reply:
Done